networkx/algorithms/mixed_edge/mixed_edge_moral.py

```diff
@@ -0,0 +1,73 @@
+import itertools
+
+import networkx as nx
+
+__all__ = ["mixed_edge_moral_graph"]
+
+
+def mixed_edge_moral_graph(
+    G,
+    directed_edge_name="directed",
+    undirected_edge_name="undirected",
+    bidirected_edge_name="bidirected",
+):
+    """Return the moral graph from an ancestral graph  in :math:`O(|V|^2)`.
+
```

Comment:
Can you add a paragraph reminding people what the moral graph is and what an ancestral graph is?
Also, you should move the complexity statement out of the first line (that is used for short text near any link in the docs to this function.  That first line should have a blank line after it. Then the paragraph.

Reply:
Done lmk wdyt.

Reply:
Yay -- progress!

Some comments on the paragraph: 
How can the undirected moral graph have a v-structure (which is directed) (it sounds like it is from the same graph)?
Is it correct that the v-structure appears in the ancestral graph? You don't actually say what the ancestral graph is -- just that it contains many types of edges. How are those edges determined (from ancestors?)
Also, the moral and ancestral graph must be related in some way, but the paragraph doesn't say how they are related. :}

Thanks!

Reply:
Okay tried to address this in a better way. Lmk if this still could be improved!